### ISSUE TITLE
This sample does not even compile

### ISSUE THREAD

**Issue by tjfan001** [Esri/arcgis-runtime-samples-dotnet#369]
Two major errors are:

1. The tag 'MapView' does not exist in XML namespace 'http://schemas.esri.com/arcgis/runtime/2013'. Line 14 Position 10.	

2. The tag 'SceneView' does not exist in XML namespace 'http://schemas.esri.com/arcgis/runtime/2013'. Line 6 Position 10.

These errors appear in many places. 

BTW, I'm using C#, not VB.

**Comment by mbranscomb** [comment]
@tjfan001 

Please can you confirm:
- What branch are you using? (should be `master`)
- What Solution / Project are you building? (e.g. `ArcGISRuntime.WPF.Viewer.sln`)

**Comment by tjfan001** [comment]
Yes, both are confirmed.

The zip file is: arcgis-runtime-samples-dotnet-master.zip. The project is ArcGISRuntime.WPF.Viewer.sln.

I'm running on VS2013 with .Net 4.6.1. When the project was first loaded, it said .Net 4.5.2 wasn't installed and I chose to use .Net 4.5. I don't think this matters. Just FYI.

It's interesting that my own project uses esri:MapView and it is compiled ok.

**Comment by tjfan001** [comment]
There are also syntax errors like this one.
![esri01](https://user-images.githubusercontent.com/34352345/33740408-67269f4a-db55-11e7-8d24-4dff094a32c7.jpg)

**Comment by mbranscomb** [comment]
@tjfan001 You are seeing the error message because you changed the project to target .NET 4.5. The minimum is 4.5.2, but you can reference any version higher than that (e.g. 4.6.1).

**Comment by tjfan001** [comment]
Thanks for the info. I did reference 4.6.1 and it removed MapView errors, but I still get the syntax error (as shown in the picture above).

Note that when the project was first loaded, it only allowed me to go back to 4.5. Afterwards, I went into the project property to change it to 4.6.1.

Did I miss anything else?

**Comment by mbranscomb** [comment]
@tjfan001 Those syntax errors are because it's using a higher version of C# than you have installed with your VS2013 installed - apologies, that's our mistake. We'll need to fix that (we should currently support all the way back to VS2012 and C# 5.0).

**Comment by tjfan001** [comment]
OK. Thanks again. 
What is that version of that C# and which Visual Studio version? I wonder if we should upgrade anyways.

**Comment by mbranscomb** [comment]
@tjfan001 If you have the option, I'd recommend updating to the latest available (VS 2017 Update 5).

**Comment by tjfan001** [comment]
OK. Thanks again. We'll think about it as upgrading sometimes can cause nightmares (we're working with current InstallShields 2014 and not sure if the transition will be smooth).

**Comment by tjfan001** [comment]
Do you have a time frame for the current sample fix so it will support Visual Studio 2012/2013? We're currently working on our project with Visual Studio 2013 and will not be able to upgrade very soon.

Thanks again for your help.

**Comment by mbranscomb** [comment]
@tjfan001 Yes - the fix is in fact already in the `v.next` branch, waiting for the release of the upcoming ArcGIS Runtime SDK v100.2 for .NET. That's imminent, so we'll be merging soon.

The fixes are in this PR (merged to v.next): https://github.com/Esri/arcgis-runtime-samples-dotnet/pull/358.

**Comment by tjfan001** [comment]
Will there be an announcement for the availability of the fix? How do I check when it is available? Please advise. Thanks.
